README.md

```diff
@@ -0,0 +1,97 @@
+
+### Markdown
+
+Blog posts are stored as [Markdown files](https://www.markdownguide.org/getting-started/) in the `/posts` directory in this repo. Our site uses a standard set a key value pairs stored at the beginning of each markdown files as `frontmatter`. Below is an example of the `frontmatter` used at the beginning of each Markdown file.
+
+```frontmatter
+---
+title: The Airflow 2.0 Scheduler
+slug: airflow-2-scheduler
+description: A technical deep-dive into Apache Airflow's refactored Scheduler, now significantly faster and ready for scale.
+heroImagePath: ../assets/scheduler-hero-2.png
+authors:
+  - Vikram Koka
+date: 2020-12-17T16:17:00.000Z
+---
+```
+
+Fill in all the fields shown above with information relative to your post. The `title` field will make a h1 tag at the top of the post with your title. Do not use h1s in your `.md` file. The `slug` field is used as the as the path to the post. The `heroImagePath` field is the path to the hero image. Hero images are stored in the `assets` directory. If you do not have a hero image use `../assets/blank.jpg`. 
+
+Images used in the blog posts are stored in the `assets` directory in this repo. Below is an example of how to include an image in your markdown folder. 
+
+```markdown
+![Airflow Scheduler Deployment Model](../assets/airflow-2-scheduler/scheduler-deployment-model.png)
+```
+
+Please follow standard markdown formatting rules that are outlined [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md) more information can be found below in the markdown Lint section.
+
+### Netlify CMS
+
+You can also use Netlify's content management system to create blog posts. The CMS tool for our blog is located at [https://astro-blog.netlify.app/admin](https://astro-blog.netlify.app/admin). When creating a post you will be prompted to provide the `frontmatter` information described above. The CMS tool will create a PR to this repo.
+
+To log into the CMS go to `/admin` and login with your github account. You will need to have access to the `astronomer/astro-blog` repo to login to the CMS. 
+
+## CI/CD Pipeline
+
+A GitHub Actions CI/CD Pipeline is used to verify each markdown file is free from formatting errors, spelling errors, and broken links. To see logs from this pipeline go to `Actions` section of the GitHub repo. Run the commands outlined below locally to fix errors before contributing posts.
+
+Fix pipeline errors from all posts as old links may break and need to be replaced.
+
+### Markdown Lint
+
+The pipeline uses [markdownlint](https://github.com/DavidAnson/markdownlint) to ensure proper markdown formatting to ensure consistency. The rules followed by the linter can be found [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md). To run the linter locally run the following.
+
+```shell
+node_modules/.bin/markdownlint ./posts/<your-blog-post> --ignore node_modules --ignore v5
```

Comment:
Seems like all of this could be baked into a Yarn script addition in `package.json`? I can't imagine this takes too long to run, so maybe just a single script that runs against the whole `./posts/` directory?

I'm guessing you don't need `--ignore node_modules` if you're not running this against the root directory.

What is `--ignore v5` referring to?

```suggestion
yarn lint
```

Reply:
I think baking all this into yarn scripts is a good idea

Reply:
forgot to remove the `--ignore` flag from testing earlier